src/utils/reducers/ShoppingCartReducer.js

```diff
@@ -0,0 +1,71 @@
+export const ShoppingStateEnums = {
+  DELETE_ITEMS: 0,
+  MINUS_ITEM: 1,
+  ADD_ITEM: 2,
+  PLUS_ITEM: 3,
```

Comment:
You are trying to create a Enum here, but them are not well seen today, in fact, when using TS this can be replaced by interfaces or const, please change it for a different way.

```suggestion
  PLUS_ITEM: 'PLUS_ITEM',
```

<img width="788" alt="Screen Shot 2023-09-18 at 10 15 47" src="https://github.com/dbracamonteswz/ondemand-react-bootcamp/assets/96157153/38fd6937-9b29-4b43-bd17-c69786c11053">

Reply:
done